test/CreateEmployeeViewController.m

```diff
@@ -26,10 +27,39 @@ - (void)viewDidLoad
 
 - (IBAction)saveEditing:(UIButton *)sender
 {
-    self.createEmployee = [[Employee alloc] initWithFirstName:self.firstNameTextField.text lastName:self.lastNameTextField.text salary:[self.salaryTextField.text intValue]];
-    
-    [self.delegate sendEmployee:self.createEmployee];
+  
+    if ((self.firstNameTextField.text.length > 0) && (self.lastNameTextField.text.length > 0) && ((self.salaryTextField.text.length > 0)))
+    {
+        Employee *createEmployee = [NSEntityDescription insertNewObjectForEntityForName:@"Employee" inManagedObjectContext:[DatabaseController sharedInstance].context];
```

Comment:
Rename createEmployee. "Create" is a verb, it is not suitable for a variable name.